subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/JavaCompile.java

```diff
@@ -141,7 +140,7 @@ public void setToolChain(JavaToolChain toolChain) {
      */
     @Deprecated
     @TaskAction
-    protected void compile(IncrementalTaskInputs inputs) {
+    protected void compile(@SuppressWarnings("deprecation") org.gradle.api.tasks.incremental.IncrementalTaskInputs  inputs) {
```

Comment:
Why use FQCN here?

Reply:
Also: double space!

Reply:
The FQDN is used her,  since I cannot suppress warnings on the import statement. Same for the other class. As for the spaces: Looks like I did too much copy and paste...